server/bin/gold/test-3.txt

```diff
@@ -409,3 +409,5 @@ run-1970-01-01T00:00:42-UTC: Total 0 files processed, with 0 md5 failures and 0
 +++ test-execution.log file contents
 ssh pbench-satellite.example.com /var/tmp/pbench-test-server/test-3/opt/pbench-server-satellite/bin/pbench-sync-package-tarballs
 --- test-execution.log file contents
++++ SqliteDB Datasets
```

Comment:
Is there a way to not emit this output if nothing was found?

Reply:
I tried several ways, and gave up; but reading this comment last night I realized there was one other thing I'd meant to try but somehow didn't get around to doing.

Reply:
> Is there a way to not emit this output if nothing was found?

I just added an extra check, but I also remembered why I'd decided not to use it before. The normal file system dump includes the file names for empty logs when it dumps the contents; so it seemed that triggering on the presence of the sqlite DB file was equivalent. (Though I had also added a check that Pbench had initialized the schema and the `dataset` table existed.) What I chose then _not_ to add, but did add this morning, is an additional `select count() from dataset` which will let me drop the DB dump entirely if no datasets were created. This is inconsistent with the log file dump behavior, although I do like it better. (And I suppose we could do the same for empty logs, since in both cases the file names already show up in the list.)